### DISCUSSION TITLE
ping available with server but no ping with wireguard client

### DISCUSSION THREAD

**Discussion by egosha17** [angristan/wireguard-install#414] [Help]
Hi,
Installed Wireguard with auto-installation script of Angristan
Platform: Linode
Server: Ubuntu 20.04 LTS

What are possible reasons?
Server pings with my phone but no ping (10.66.66.2) over wireguard.
Location: Turkmenitan (ISP is good at blocking VPNs. They can even block OpenVPN)
So I want to practice Wireguard but could not accomplished yet.

root@localhost:~# wg
interface: wg0
  public key: lv23qtp+QeA+4vhQjoTAyhxy3TsXxRBf0szDiK236V8=
  private key: (hidden)
  listening port: 443

peer: c1bqF6pgdPdcwjwLMKPUAX1rt6vXajL2cAYkz+uYOjc=
  preshared key: (hidden)
  endpoint: 95.85.101.XXX:19225
  allowed ips: 10.66.66.2/32, fd42:42:42::2/128
  transfer: 296 B received, 12.61 KiB sent

root@localhost:~# ping 10.66.66.2
PING 10.66.66.2 (10.66.66.2) 56(84) bytes of data.

--- 10.66.66.2 ping statistics ---
429 packets transmitted, 0 received, 100% packet loss, time 438263ms

CLIENT
[Interface]
PrivateKey = mIXVxUkziMaowDmHMCadQZgYZK83GV3jN+andHcQ304=
Address = 10.66.66.2/32,fd42:42:42::2/128
DNS = 94.140.14.14,94.140.15.15

[Peer]
PublicKey = lv23qtp+QeA+4vhQjoTAyhxy3TsXxRBf0szDiK236V8=
PresharedKey = alc5kPeiEp40zSlCLZBAQIp0/VcjbgysJrmZr1HvGTg=
Endpoint = 45.79.37.XX:443
AllowedIPs = 0.0.0.0/0,::/0

443 port connects with OpenVPN both TCP&UDP,

**Comment by CommandCracker8** [comment]
I can't help but I'd recommend not sharing any of your keys :P
